File: THD.Domain/Models/AccountModels/Request/UpdateUserRequest.cs
using System.ComponentModel.DataAnnotations;
using THD.Domain.Models.Enums;

namespace THD.Domain.Models.AccountModels.Request
{
    public class UpdateUserRequest
    {
        [Required]
        public string Id { get; set; }
        [Required]
        public string UserName { get; set; }
        [Required]
        public string Email { get; set; }
        [Required]
        public string FirstName { get; set; }
        [Required]
        public string LastName { get; set; }
        [Required]
        public string Password { get; set; }
        [Required]
        public UserRoleModel UserRole { get; set; }

    }
}

File: THD.WEB/app/src/app/swagger/shared/models/update-user-request.ts
/* tslint:disable */
import { UserRoleModel } from './user-role-model';
export interface UpdateUserRequest {
  id: string;
  userName: string;
  email: string;
  firstName: string;
  lastName: string;
  password: string;
  userRole: UserRoleModel;
}
